### PR TITLE
Adds TestViewOps to test suite

### PR DESCRIPTION
**Pull request by mruberry**
Per title. Five failing tests are disabled. These are:

- test_contiguous_nonview
- test_expand_as_view
- test_expand_view
- test_reshape_nonview
- test_unfold_view

### CODE REVIEW

File: test/run_tests.sh [hunk 1]
```diff
@@ -40,13 +40,14 @@ function run_dynamic {
 }
 
 function run_all_tests {
+  run_dynamic python3 "$CDIR/../../test/test_torch.py" "$@" -v TestViewOpsXLA
   python3 "$CDIR/../../test/test_torch.py" "$@" -v TestTorchDeviceTypeXLA
   run_dynamic python3 "$CDIR/../../test/test_torch.py" "$@" -v TestDevicePrecisionXLA
   python3 "$CDIR/../../test/test_torch.py" "$@" -v TestTensorDeviceOpsXLA
   python3 "$CDIR/../../test/test_indexing.py" "$@" -v TestIndexingXLA
   python3 "$CDIR/../../test/test_indexing.py" "$@" -v NumpyTestsXLA
-  run_dynamic python3 "$CDIR/../../test/test_nn.py" "$@" -v TestNNDeviceTypeXLA
```
Comment (dlibenzi) [inline test/run_tests.sh]: Why this change?

**Comment by dlibenzi** [comment]
You need to rebase @mruberry

File: test/run_tests.sh (hunk 1 above)
Comment (mruberry) [reply test/run_tests.sh]: This test fails on our DevFairs so I moved it after TestTypePromotion so we could run more of the test suite easily.

Comment (dlibenzi) [reply test/run_tests.sh]: I don't understand. If tests fail we should be fixing them or blacklisting them.
This change does not seem to below to this PR.
